.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,3 @@
-## Read and complete the full issue template
-
 **Do you want to request a *feature* or report a *bug*?**
```

Comment:
This file should remain untouched. I don't really understand how it got changed, but please restore the version that is currently in the `ClosedXML` repository.

Reply:
Sorry, I restore from currently ClosedXML repository